src/pages/instances/InstanceOverviewMetrics.tsx

```diff
@@ -32,7 +36,7 @@ const InstanceOverviewMetrics: FC<Props> = ({ instance, onFailure }) => {
 
   return (
     <>
-      {isLoading ? (
+      {isLoading && !isRestricted ? (
```

Comment:
I think this is wrong, UX-wise. If I understand correctly what's going on here, the restricted user would get shown the empty "-" placeholder. So, in principle, one cannot really distinguish whether they're seeing a "-" placeholder because they are restricted, or because the metrics are not available (e.g. the instance is not running).
A better UX is to clearly show the user that this data (the metrics) is not accessible due to an existing restriction, when this is the case.

Reply:
Yes, we can show a dedicated message for restricted users, changed it accordingly.